culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+This post proposes a system of **trust levels** for SourceCred the
+system, which describe how we intend to handle the very real questions
+of adversarial actors and gaming resistance over different time scales.
+
```

Comment:
Has it been referred to as "SourceCred the system?" What exactly is a system here - an implementation, or an interface, or something else? I think it would read cleanly to remove the system (and people know we are talking about SourceCred) or to be more specific about what "the system" is. Am I entering the matrix? My web browser?